medusa/logger/__init__.py

```diff
@@ -219,34 +219,23 @@ def blocks_r(filename, size=64 * 1024, reset_offset=True, encoding='utf-8'):
         size (int|None): The block size.
```

Comment:
Size cannot be `None` anymore, applies to both functions,
it would just cause an exception, because
```py
# remaining_size = 12345
# size = None
block_size = min(remaining_size, size)
# block_size = None
file_obj.seek(remaining_size - block_size)
# Exception
```